pom.xml

```diff
@@ -1519,7 +1494,6 @@
       </activation>
       <modules>
         <module>phoenix-client-parent</module>
-        <module>phoenix-client-parent/phoenix-client</module>
```

Comment:
Drop `phoenix-client-parent/phoenix-client` directory and pom.xml too?

Reply:
We could. 
However, I plan to have at least one more variant that that can coexist  hbase-shaded-client (PHOENIX-6053), and it's easier to keep the structure than to remove and re-add it later.
If you feel strongly about it, then we certainly can remove this.

Reply:
Oh, you mean the directory.
Sure.

Reply:
Done.